src/main/kotlin/nl/info/zac/authentication/UserPrincipalFilter.kt

```diff
@@ -40,11 +40,18 @@ constructor(
     private val zaakafhandelParameterService: ZaakafhandelParameterService,
     private val pabcClientService: PabcClientService,
     @ConfigProperty(name = "FEATURE_FLAG_PABC_INTEGRATION", defaultValue = "false")
-    private val pabcIntegrationEnabled: Boolean
+    private val pabcIntegrationEnabled: Boolean,
 ) : Filter {
     companion object {
         private val LOG = Logger.getLogger(UserPrincipalFilter::class.java.name)
         private const val GROUP_MEMBERSHIP_CLAIM_NAME = "group_membership"
+        val pabcDefinedRoles: Set<String> = setOf(
```

Comment:
```suggestion
        val SUPPORTED_PABC_FUNCTIONAL_ROLES: Set<String> = setOf(
```

This is the set of functional roles for which we currently provide mappings in the PABC. This is a very temporary situation only because of the current behaviour in PABC and this filtering will be removed once PABC allows requesting mappings for functional roles for which there are now mappings. Good to add this as a comment.